fuzz/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "snarkos-fuzz"
+version = "0.0.0"
+authors = ["Automatically generated"]
+publish = false
+edition = "2018"
```

Comment:
Why not `2021`?

Reply:
it was auto-generated; I guess I can bump it to 2021